vhdbuilder/packer/install-dependencies.sh

```diff
@@ -448,9 +448,9 @@ rm -r /var/log/azure/Microsoft.Azure.Extensions.CustomScript || exit 1
 # So keeping as many versions as we can - those unsupported version can be removed when we don't have enough space
 # NOTE that we keep multiple files per k8s patch version as kubeproxy version is decided by CCP.
 
-# kube-proxy regular versions >=v1.17.0  hotfixes versions >= 20211009 are 'multi-arch'. All versions in kube-proxy-images.json are 'multi-arch' version now.
+# kube-proxy regular versions >=v1.17.0  hotfixes versions >= 20211009 are 'multi-arch'. All versions in manifest.json are 'multi-arch' version now.
 
-KUBE_PROXY_IMAGE_VERSIONS=$(jq -r '.containerdKubeProxyImages.ContainerImages[0].multiArchVersions[]' <"$THIS_DIR/kube-proxy-images.json")
+KUBE_PROXY_IMAGE_VERSIONS=$(jq -r '."kube-proxy".containerdKubeProxyImages.ContainerImages[0].multiArchVersions[]' <"$THIS_DIR/manifest.json")
```

Comment:
maybe we dint need the double quote if they are removed from the key

Reply:
I had to wrap it around double quote due to jq issue with dashed key